nservicebus/azure-service-bus/multiple-namespaces-support.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Multiple namespace support
+summary: Configuring Azure ServiceBus transport to support different partitioning strategies. 
+component: ASB
+tags:
+- Cloud
+- Azure
+- Transports 
+reviewed: 2016-04-18
+---
+
+`Azure Service Bus` transport enables different _namespace partitioning strategies_ to cover different scenarios such as High Availability and multiple Data Center support or to overcome [service limits](https://azure.microsoft.com/en-us/documentation/articles/service-bus-quotas/).  
+
+Configuration APIs give the possibility to select desired _namespace partitioning strategies_ and then to configure specific namespace(s) to use.
```

Comment:
If I read correctly, this is the intent?

"_Namespace partitioning strategy_ can be configured using Azure Service Bus configuration API, specifying namespace(s) to use."